### PR TITLE
env var COOKIE_VALIDATION_KEY + fix assets mode

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docker-compose-example.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ services:
             ENABLE_ENV_FILE: 1
             ENABLE_LOCALCONF: 1
             API_TOKEN: "<YOUR GITHUB API TOKEN>"
+            COOKIE_VALIDATION_KEY: "<YOUR COOKIE VALIDATION KEY>"
```
Comment (mikehaertl) [inline docker-compose-example.yml:14]: This variable is already set in `.env`. We don't want to duplicate all app configuration vars in the dockerfile.

File: Dockerfile [hunk 2]
```diff
@@ -13,6 +13,8 @@ RUN composer self-update --no-progress && \
 # Copy the working dir to the image's web root
 COPY . /var/www/html
 
+RUN chmod a+rwx /var/www/html/assets
```
Comment (mikehaertl) [inline Dockerfile:16]: What is this for? The assets directory for the assetManager is created below with correct permissions.

Comment (ghost) [reply Dockerfile:16]: There were an errors in the log

Comment (mikehaertl) [reply Dockerfile:16]: What was the error? You should never have to write to this directory. The correct assets directory is `/var/www/html/web/assets`. And that directory should already have the correct permissions.

**Comment by mikehaertl** [comment]
@ev4ern If you think there's still a problem, please open an issue and provide the steps to reproduce the error together with all relevant information about OS, versions, etc.

**Pull request closed without merging**